test/api/auth.js

```diff
@@ -73,4 +73,91 @@ describe('Authenticate', () => {
       });
     });
   });
+
+  describe('Login', () => {
+    before((done) => {
+      const user = new User({
+        name: 'kissa andela',
+        username: 'jjjk',
+        email: 'kissa@andela',
+        password: '123456',
+      });
+      user.save((error) => {
+        if (error) {
+          throw error;
+        }
+        done();
+      });
+    });
+
+    it('Should return an error on wrong email address', (done) => {
+      const user = {
+        email: 'kissass@andela',
+        password: '123456'
+      };
+      chai.request(server)
+      .post('/api/auth/login')
+      .send(user)
+      .end((error, response) => {
+        response.should.have.status(401);
+        response.body.should.have.property('message');
+        response.body.should.have.property('success').eql(false);
```

Comment:
Nice work updating this 😉 

You can remove the check 
```javascript
response.body.should.have.property('message');
```
since this test below already covers that and would fail if the `message` property is not present.
```javascript
response.body.should.have.property('message').eql('Authentication failed');
```